src/app/pages/auth/components/consent-page-item/consent-page-item.component.scss

```diff
@@ -16,7 +16,8 @@ page-consent-item {
   }
 
   .text-justify {
```

Comment:
Can you change the css class name to text-align as well. This is slightly misleading.

Reply:
Ok.